server.js

```diff
@@ -110,28 +128,21 @@ api.get(/^\/products$/, (_, res) => res.redirect("textile/products"));
 const cleanRedirect = (url) => (url.startsWith("/") ? url : "");
 api.get(/^\/simulator(.*)$/, ({ url }, res) => res.redirect(`/api/textile${cleanRedirect(url)}`));
 
-const getProcesses = async (token) => {
-  let headers = {};
-  if (token) {
-    headers["token"] = token;
-  }
+const getProcesses = async (encrypted = true) => {
+  const { ENCRYPTION_KEY } = process.env;
 
-  const processesUrl = `http://127.0.0.1:${django_port}/processes/processes.json`;
-  const processesRes = await fetch(processesUrl, { headers: headers });
-  const processes = await processesRes.json();
-  return { processes: processes, status: processesRes.status };
+  if (encrypted) {
+    return encrypt(JSON.stringify(processes), ENCRYPTION_KEY);
```

Comment:
Even this could be computed before server starts, provided `ENCRYPTION_KEY` is never dynamically updated at runtime